src/targeting/unsure.ts

```diff
@@ -0,0 +1,56 @@
+/* -- Types -- */
+
+/**
+ * I think all of these are deprecated?!?
+ */
+export type UnsureTargeting = {
+	/**
+	 * **G**uar**d**ia**n** **C**ustomer **R**elation **M**anagement – [see on Ad Manager][gam]
+	 *
+	 * Type: _Dynamic_
+	 *
+	 * Sample values:
+	 * - `"PXPE"`
+	 * - `"PXHCI2"`
+	 *
+	 * [gam]: https://admanager.google.com/59666047#inventory/custom_targeting/detail/custom_key_id=184047
+	 *
+	 * @deprecated (?)
+	 */
+	gdncrm: string | string[];
+
+	/**
+	 * **M**edia **S**ource – [see on Ad Manager][gam]
+	 *
+	 * Type: _Dynamic_
+	 *
+	 * Sample values:
+	 * - `"epictv"`
+	 * - `"crane.tv"`
+	 * - `"a-rational-fear"`
+	 *
+	 * [gam]: https://admanager.google.com/59666047#inventory/custom_targeting/detail/custom_key_id=186807
+	 *
+	 * @deprecated - We no longer use Video.js (?)
+	 */
+	ms: string;
+
+	/**
+	 * Ad **Slot** ID – [see on Ad Manager][gam]
+	 *
+	 * Type: _Predefined_
+	 *
+	 * Sample values: 'top-above-nav', 'inline-1'
+	 *
+	 * [gam]: https://admanager.google.com/59666047#inventory/custom_targeting/detail/custom_key_id=174447
+	 */
+	slot: string; // TODO: Narrow to known IDs
+};
+
+/* -- Methods -- */
+
+/* -- Targeting -- */
+
+export const getUnsureTargeting = (
+	targeting: UnsureTargeting,
```

Comment:
Have we thought about creating a log when any of these properties are used in order to confirm whether or not they're deprecated?

Reply:
Yes: https://docs.google.com/spreadsheets/d/10vsny8P_hdtBSNBp3DSRpRV49e_mVJ24Q43NY4askeY/

Reply:
Happy with removing `gdrm` & `ms`. Just checking that `slot` will only be removed from the [BuildPageTargeting](https://github.com/guardian/frontend/blob/8bf15042f04c15875afbba5c5f93fcfa979e4357/static/src/javascripts/projects/common/modules/commercial/build-page-targeting.ts#L289)?

Reply:
Yes, only from the page targeting, not for individual slots.